src/main/java/cms/gov/madie/measure/resources/ExportController.java

```diff
@@ -136,13 +138,23 @@ public ResponseEntity<byte[]> getQRDA(
       throw new ResourceNotFoundException("Measure", id);
     }
 
-    return ResponseEntity.status(HttpStatus.OK)
+    try {
+      qrdaPackage = exportService.getQRDA(requestDTO, accessToken);
+      log.info("QRDA export successful for measure [{}] by user [{}]", id, username);
+    } catch (Exception e) {
+      log.error("Failed to export QRDA for measure [{}] by user [{}]", id, username, e);
+      throw e;
+    }
```

Comment:
don't think you need to redo the error handling. its already done at service level.